app/views/_user_util_links.html.erb

```diff
@@ -13,12 +13,15 @@ Unless required by applicable law or agreed to in writing, software distributed
   specific language governing permissions and limitations under the License.
 ---  END LICENSE_HEADER BLOCK  ---
 %>
-<div class="navbar-left">
-  <ul class="nav navbar-nav">
+<div class="container-fluid grey">
+<div class="mast-branding hidden-xs">
+  <ul class="col-lg-8 col-lg-offset-2">
+  
     <% unless current_page?(root_url) %>
-    <li><%= link_to 'Home', root_path %></li>
+ 
+    <li><%= link_to 'home', root_path %></li>
     <% end %>
-    <li><%= link_to 'Browse', catalog_index_path(search_field: 'all_fields', utf8: '✓', q: '') %></li>
+    <li><%= link_to 'browse', catalog_index_path(search_field: 'all_fields', utf8: '✓', q: '') %></li>
```

Comment:
I just copied this from the existing code:

https://github.com/ualbertalib/avalon/blob/master/app/views/_user_util_links.html.erb